docs/dictionary/property/fullscreenmode.lcdoc

```diff
@@ -35,6 +35,24 @@ end mouseUp
 Value:
 The <fullscreenmode> returns the mode to which this property is set.
 
+- empty - The stack is resized (not scaled) to fit the screen. (default)
+ This is the existing behavior.
```

Comment:
I think this line ("This is the existing behaviour") can probably removed, since it doesn't really make sense in this context.